README.md

```diff
@@ -43,6 +43,13 @@ After a successful build, you'll have a new folder ```public``` in the repositor
 ```
 It will automatically use ```public``` as base directory, go to ```localhost:8080``` to view the website.
 
+### Testing
+[Cypress](https://docs.cypress.io/guides/overview/why-cypress.html#In-a-nutshell) is used to run End-To-End tests. tests are located in the ```cypress/integration``` folder and can be run with:
+```bash
+  npm test
```

Comment:
Running npm tests seems to create folder cypress/integration, cypress/videso and cypress/support. Shall these be added to the to the .gitignore?

Reply:
cypress/integration containing the test cases, so this definitiv not. The following can be added to gitignore:

- cypress/plugins (currently no plugins needed)
- cypress/screenshots (screenshot from failed tests)
- cypress/support (currently no support code needed)
- cypress/videos (video from failed tests)

I will ad an commit for this